pkg/gjson/row.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/stackrox/rox/pkg/errox"
 	"github.com/tidwall/gjson"
+	"k8s.io/utils/strings/slices"
```

Comment:
Why not using Contains from stdlib https://pkg.go.dev/slices#Contains

Reply:
👍